README.md

```diff
@@ -63,7 +63,7 @@ To run nginx proxy as a separate container you'll need:
 curl https://raw.githubusercontent.com/jwilder/nginx-proxy/master/nginx.tmpl > /path/to/nginx.tmpl
 ```
 
-2) Set the `NGINX_DOCKER_GEN_CONTAINER` environment variable to the name or id of the docker-gen container.
+2) Use the `com.github.jrcs.letsencrypt_nginx_proxy_companion.docker_gen=true` label on the docker-gen container, or explicitly set the `NGINX_DOCKER_GEN_CONTAINER` environment variable to the name or id of that container.
```

Comment:
It's better for performance but also makes it easier to filter by simply a label or label=value. 

I improved my [docker-label-sighup](https://github.com/helderco/docker-gen/blob/master/docker-label-sighup), based on this. 

It's cooler now, because it allows me to do this: `-notify "docker-label-sighup com.docker.swarm.service.name=frontend_nginx"`

So I can reuse the labels that already exist.